MapleServer2/Packets/LoginResultPacket.cs

```diff
@@ -24,21 +60,29 @@ public static Packet InitLogin(long accountId)
             return pWriter;
         }
 
-        // Set 0 for all values in login packet
-        public static Packet Timeout(long accountId)
+        public static Packet IncorrectId()
         {
             PacketWriter pWriter = PacketWriter.Of(SendOp.LOGIN_RESULT);
-            pWriter.WriteByte(0x19); // Login State
-            pWriter.WriteInt(); // Const
-            pWriter.WriteUnicodeString(""); // Ban reason
-            pWriter.WriteLong(accountId);
-            pWriter.WriteLong(); // SyncTime
-            pWriter.WriteInt(); // SyncTicks
-            pWriter.WriteByte(); // TimeZone
-            pWriter.WriteByte(); // BlockType
-            pWriter.WriteInt(); // Const
-            pWriter.WriteLong(); // Const
-            pWriter.WriteInt(); // Const]
+            pWriter.WriteEnum(LoginModes.IncorrectId);
+            pWriter.WriteZero(45);
```

Comment:
Why was the structure of the packet removed?

Reply:
it's supposed to be all zeros.